tc/sgx/trusted_worker_manager/enclave/enclave_data.h

```diff
@@ -65,7 +69,11 @@ class EnclaveData {
     // allocated storage for the sealed data. There are a number of ways
     // to do this; for now, the constant will be good enough.
     static const size_t cMaxSealedDataSize = 8192;
-    static const size_t cMaxPublicDataSize = 4096;
+    /*
+       size of the base64 encoded encryption key signature is 2733
+       formula to calculate size - (no of bytes * 4 / 3) + 2 + 1
+    */
+    static const size_t cMaxPublicDataSize = 6829;
```

Comment:
calculate the size programatically based on above comment instead of hard coding the value. Also calculate the base64 encoded encryption key signature.  Update the comment to be more elaborate like why are we adding 2+1 in calculation above (mention about double "=" for padding and extra byte to store null character).

Reply:
the encryption_key_signature variable stores the base64 encoded version. I will add more description to the formula.